manifests/init.pp

```diff
@@ -72,6 +89,32 @@
     }
   }
 
+  if $manage_crontab {
+    # Template uses:
+    # - $crontab_shell
+    # - $crontab_path
+    # - $crontab_mailto
+    # - $crontab_home
+    # - $crontab_run_parts
+    file { '/etc/crontab':
+      ensure  => file,
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0644',
```

Comment:
IMO all these `file` defaults should be left unspecified since they are defaults. Specifying the owner and group like this causes portability problems for BSDs and macOS because they use group `wheel` instead of `root`.

Reply:
I have concerns with that because that allows configuration drift.  If the state of `owner` and `group` are not managed by Puppet then it's possible for an admin to put a system into a state where those files have the incorrect ownership and then Puppet would not put the values to their correct values.  If those values have different values on different OSes then the Hiera data in the module just needs to have appropriate defaults for those operating systems to avoid leaving things unmanaged by Puppet.  Also there are already other files in this module with `owner => 'root'` and `group => 0` so if those should already be causing problems if those files are enabled.  Also that file is opt-in so by default that file is not even managed.

Reply:
I changed this pull request's file resources to `group => 0` to match other things which on my Mac belongs to `wheel` so that should solve that problem and match what other files in this module are doing.